pages/common/bzip3.md

```diff
@@ -7,11 +7,11 @@
 
 `bzip3 {{path/to/file_to_compress}}`
 
-- Decompress a file:
+- [d]ecompress a file:
 
 `bzip3 -d {{path/to/compressed_file.bz3}}`
 
-- Decompress a file to `stdout`:
+- Decompress a file to `stdout` ([c]):
```

Comment:
Is this kind of mnemonic usage documented? I can't find anything in the style guide.
```suggestion
- Decompress a file to `stdout`:
```

Reply:
Nope, I've seen this in some pages, but we can document this.

Reply:
Remember that the purpose of the guides are to **describe** our practices, not to **prescribe** them.

The style guide doesn't say anything about this. In my opinion, mnemonics between parenthesis are nice when there are multiple options in the example command.

Reply:
> Is this kind of mnemonic usage documented?

It isn't documented but we use this in a few translations of popular commands where we need to document the short option or context.

Reply:
I will try to create a PR to document this in the style guide later today.